src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicApi.cs

```diff
@@ -116,6 +120,16 @@ static MsQuicApi()
                 return;
             }
 
+            if (isWindows && IsTls1_3Disabled())
```

Comment:
It also seems like there is already check for isWindows above. I would probably embed new code  and `IsWindowsVersionSupported` into it.